README.md

```diff
@@ -88,7 +98,7 @@ What about `javascript`?
 - `framework`
 The deep learning framework of the source code. For now, we support `pytorch` and `tensorflow`.
 Can be `null` if the implementation is not framework specific.\
-`language` and `framework` define which model runner can use this model for inference. 
+`language` and `framework` define which model runner can use this model for inference (see [supported_formats_and_operations.md#preprocessing](https://github.com/bioimage-io/configuration/blob/master/supported_formats_and_operations.md#preprocessing)). 
```

Comment:
This part of the spec is actually not quite correct, and I proposed changes to it in #50.

Reply:
I am not sure how to best combine these two PRs, maybe it's best if we merge this one and then just rebase #50 on top of it.

Reply:
Sorry, I missed that one. Then, in the `weights` or `weight_format` definition, I'd indicate that this field is what determines `which model runner can load and use this model for inference`. I'd say that this is a key detail for future contributors. For example:
```yaml
- weight_format` Format of this set of weights. Each model runner is limited to a specific format. Weight formats can define additional (optional or required) fields. See [supported_formats_and_operations.md#Weight Format](https://github.com/bioimage-io/configuration/blob/master/supported_formats_and_operations.md#weight_format)
```

> I am not sure how to best combine these two PRs, maybe it's best if we merge this one and then just rebase #50 on top of it.

Go on with the easiest way for you. I can also remove this PR and do it on a different branch.

Reply:
> Sorry, I missed that one. Then, in the `weights` or `weight_format` definition, I'd indicate that this field is what determines `which model runner can load and use this model for inference`

Yes, I fully agree. We should add this to `weights` or `weight_format`.

> Go on with the easiest way for you. I can also remove this PR and do it on a different branch.

Let's go ahead with this one and I will rebase #50 later.